src/app/components/searchBox/search-box.component.spec.ts

```diff
@@ -0,0 +1,107 @@
+
```

Comment:
Too many imports. 
Have you considered creating your own material.module.ts file?
This will only import and then export the modules specific to the application